manage.py

```diff
@@ -87,52 +94,41 @@ def seed_db():
             db.session.add(housekeeping)
     db.session.commit()
 
-    # TODO: import groundStation commands (apps, services) instead of hardcoding
-    commands = {
-        'ping': (0, False),  # (number of args, is_dangerous)
-        'get-hk': (0, False),
-        'turn-on': (1, True),
-        'turn-off': (1, True),
-        'upload-fs': (0, False),
-        'adjust-attitude': (1, True),
-        'magnetometer': (0, False),
-        'imaging': (0, False),
-        'demo.time_management.set_time': (1, False),
-        'demo.time_management.get_time': (0, False)
-    }
-
+    commands = import_commands() # empty dictionary if import fails
     for name, (num_args, is_danger) in commands.items():
         c = add_telecommand(command_name=name,
                             num_arguments=num_args, is_dangerous=is_danger)
```

Comment:
Try putting the entire process of adding gs_software commands (lines 97 to 100) inside the import_commands() function as we would like to be able to import new commands while the server is running without needing to re-seed the entire db. While the server is running, we could instead run something like `python3 manage.py import_commands` to fetch new commands from gs_software when new ones are added.  

We may also need to modify [`add_telecommand`](https://github.com/AlbertaSat/ex2_ground_station_website/blob/60fc0bd640046ae25b9240146e1b48ac22ba4822/groundstation/backend_api/utils.py#L33) to update the telecommands table if the command we're trying to add already exists.

Reply:
Yea lets do this - just tweak add_telecommand to only insert if the command does not already exist.